internal/pkg/flink/internal/store/store_test.go

```diff
@@ -363,32 +363,66 @@ func (s *StoreTestSuite) TestParseSETStatement() {
 	key, value, _ = parseSetStatement("set 'key= \nvalue'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+
+	key, value, _ = parseSetStatement(`set ''key''=''value''`)
+	assert.Equal(s.T(), "", key)
+	assert.Equal(s.T(), "", value)
+
+	key, value, _ = parseSetStatement(`set '"key"'='"value"'`)
+	assert.Equal(s.T(), `"key"`, key)
+	assert.Equal(s.T(), `"value"`, value)
 }
 
-func (s *StoreTestSuite) TestParseSETStatementerror() {
+func (s *StoreTestSuite) TestParseSETStatementError() {
 	_, _, err := parseSetStatement("SET key")
-	assert.NotNil(s.T(), err)
-	assert.Equal(s.T(), "Error: missing \"=\"\nUsage: \"SET 'key'='value'\"", err.Error())
+	assert.Equal(s.T(), &types.StatementError{
+		Message: `missing "="`,
+		Usage:   []string{"SET 'key'='value'"},
+	}, err)
 
 	_, _, err = parseSetStatement("SET =")
-	assert.NotNil(s.T(), err)
-	assert.Equal(s.T(), "Error: key and value not present\nUsage: \"SET 'key'='value'\"", err.Error())
+	assert.Equal(s.T(), &types.StatementError{
+		Message: "key and value not present",
+		Usage:   []string{"SET 'key'='value'"},
+	}, err)
 
 	_, _, err = parseSetStatement("SET key=")
-	assert.NotNil(s.T(), err)
-	assert.Equal(s.T(), "Error: value for key not present\nSuggestion: if you want to reset a key, use \"RESET 'key'\"", err.Error())
+	assert.Equal(s.T(), &types.StatementError{
+		Message:    "value for key not present",
+		Suggestion: `if you want to reset a key, use "RESET 'key'"`,
+	}, err)
 
 	_, _, err = parseSetStatement("SET =value")
-	assert.NotNil(s.T(), err)
-	assert.Equal(s.T(), "Error: key not present\nUsage: \"SET 'key'='value'\"", err.Error())
+	assert.Equal(s.T(), &types.StatementError{
+		Message: "key not present",
+		Usage:   []string{"SET 'key'='value'"},
+	}, err)
 
 	_, _, err = parseSetStatement("SET ass=value=as")
```

Comment:
:eyes:

Reply:
Haha oops, I guess that's an unfortunate typo. Will rename!